package.json

```diff
@@ -3,7 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "scripts": {},
+  "scripts": {
+    "allTests-chromium": "npx playwright test --project=chromium --workers=6",
+    "allTests-firefox": "npx playwright test --project=firefox --workers=6"
```

Comment:
There is no point of forcing Playwright to use more workers.
Playwright automatically use the maximum number of workers based on how many logical CPU cores you have in your processor. Playwright uses half of CPUs to run the tests, so if your computer does not have 12 or more cores, this setting will not do anything. Usually this setting used to limit the number of workers, for example, use no more than 2-3.